packages/sui-studio/package.json

```diff
@@ -14,7 +14,7 @@
     "@s-ui/helpers": "1",
     "@s-ui/react-context": "1",
     "@s-ui/react-router": "1",
-    "@s-ui/test": "8",
+    "@s-ui/test": "no-coverage-inline",
```

Comment:
I think this is off 🤔

Reply:
Yes, but I needed to do a real test in a GHA execution... I'll remove it before merging it ;)